com.github.aharotias2.parapara.yml

```
@@ -0,0 +1,23 @@
+app-id: com.github.aharotias2.parapara
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.aharotias2.parapara
+finish-args:
+  - '--share=ipc'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--filesystem=home'
+modules:
+  - name: granite
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/elementary/granite.git
+        tag: '6.2.0'
+        commit: 4ab145c28bb3db6372fe519e8bd79c645edfcda3
+  - name: parapara
+    buildsystem: meson
+    sources:
+      - type: dir
+        path: .
```

```suggestion
      - type: git
        url: https://github.com/aharotias2/parapara.git
        tag: 'v3.2.1'
        commit: ecc80af99ca8d7fc2811a585f85ba07f7e065fc8
```

FYI I added a diff. And every time you create a new release in your repository replace `tag` and `commit` values.

Note that this changes should be only for releasing for Flathub so you shouldn't modify the original yaml file in your repository which is for development.